src/kata5.reachDestination.js

```diff
@@ -1,3 +1,8 @@
-const reachDestination = (distance, speed) => {};
+const reachDestination = (distance, speed) => {
+    const time = distance/speed
+    const decimal = time - Math.floor(time)
+    return `I should be there in ${decimal < 0.75 && decimal > 0.25? Math.floor(time)+0.5:Math.round(time)}hours.`
```

Comment:
Nice! I would make sure to add a whitespace right before hours though so the answer is more readable:
```
    return `I should be there in ${decimal < 0.75 && decimal > 0.25? Math.floor(time)+0.5:Math.round(time)} hours.`
```